glean-core/src/database/mod.rs

```diff
@@ -537,13 +543,42 @@ impl Database {
             .write()
             .expect("Can't access app lifetime data as writable")
             .clear();
+    }
 
+    /// Persist ping_lifetime_data to disk and clear it.
+    ///
+    /// Does nothing in case there is nothing to persist.
+    ///
+    /// ## Panics
+    ///
+    /// * This function will **not** panic on database errors.
+    pub fn persist_ping_lifetime_data(&self) -> Result<()> {
```

Comment:
I think this needs to be exposed somehow for external consumers. Where are you planning to expose it? Is that part of a different PR?

Reply:
So, I wasn't sure just adding it as a public method to the Database module was enough.

I don't think it needs to be a different PR. Should I create a method in the Glean object to access this functionality?

Reply:
We should probably add it to both the Glean Rust object and then over FFI to the Glean object General APIs for each language we support and for which we exposed the option in #530

Reply:
Makes sense!

Reply:
Okay, as discussed AFK, I believe this should not be added to the **android** and **ios** bindings, as it is desktop specific. I will add it to the Rust object for now and maybe to the python bindings? What do you think?

I will also remove the options to defer ping lifetime data persistence from those platforms.

Does that sound good?